[Bug] Travis CI build fails with invalid syntax for f-string in Python 3.5

**Describe the bug**

Travis CI build fails with SyntaxError for an f-string.

**To Reproduce**

See builds for the [releasebranch_7_8](https://github.com/OSGeo/grass/tree/releasebranch_7_8) branch.

**Expected behavior**

I don't know. The CI runs with 16.04. Should we update the Travis config? Clearly, it is first compilation-related Python >=3.6 requirement. Perhaps, we do want to keep the Python requirements low in 7.8? I would say, at least update whatever the commit is breaking.

**Screenshots**

```
LD_LIBRARY_PATH="/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/bin:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/bin:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/scripts:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/lib:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/lib:" LC_ALL=C LANG=C LANGUAGE=C  --html-description < /dev/null | grep -v '</body>\|</html>' > pngdriver.tmp.html ; fi
VERSION_NUMBER=7.8.7dev VERSION_DATE=2022 MODULE_TOPDIR=../.. \
        python3 /home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/tools/mkhtml.py pngdriver > /home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/docs/html/pngdriver.html
  File "/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/tools/mkhtml.py", line 84
    trunk_url = f"https://github.com/OSGeo/grass/tree/releasebranch_{major}_{minor}/"
                                                                                    ^
SyntaxError: invalid syntax
../../include/Make/Html.make:7: recipe for target '/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/docs/html/pngdriver.html' failed
make[4]: *** [/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/docs/html/pngdriver.html] Error 1
make[4]: Leaving directory '/home/travis/build/OSGeo/grass/lib/pngdriver'
../../include/Make/Lib.make:26: recipe for target 'lib' failed
make[3]: *** [lib] Error 2
```

```
make[4]: Entering directory '/home/travis/build/OSGeo/grass/lib/pngdriver'
if [ "" != "" ] ; then GISRC=/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/demolocation/.grassrc78 GISBASE=/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu PATH="/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/bin:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/bin:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/scripts:$PATH" PYTHONPATH="/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/etc/python:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/gui/wxpython:$PYTHONPATH" LD_LIBRARY_PATH="/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/bin:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/bin:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/scripts:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/lib:/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/lib:" LC_ALL=C LANG=C LANGUAGE=C  --html-description < /dev/null | grep -v '</body>\|</html>' > pngdriver.tmp.html ; fi
VERSION_NUMBER=7.8.7dev VERSION_DATE=2021 MODULE_TOPDIR=../.. \
        python3 /home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/tools/mkhtml.py pngdriver > /home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/docs/html/pngdriver.html
  File "/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/tools/mkhtml.py", line 85
    addons_url = f"https://github.com/OSGeo/grass-addons/tree/grass{major}/"
                                                                           ^
SyntaxError: invalid syntax
../../include/Make/Html.make:7: recipe for target '/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/docs/html/pngdriver.html' failed
make[4]: *** [/home/travis/build/OSGeo/grass/dist.x86_64-pc-linux-gnu/docs/html/pngdriver.html] Error 1
make[4]: Leaving directory '/home/travis/build/OSGeo/grass/lib/pngdriver'
../../include/Make/Lib.make:26: recipe for target 'lib' failed
```


**System description**

```
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.7 LTS
Release:	16.04
Codename:	xenial
...
python3 is already the newest version (3.5.1-3).
python3-dev is already the newest version (3.5.1-3).
python3-six is already the newest version (1.10.0-3).
```

**Additional context**

Fails since cda422017b99c4a72bb632505c917a8d49c587ee (#2037) which introduced the first f-string (some more f-strings appeared in the code since then).

--- a/tools/mkhtml.py
+++ b/tools/mkhtml.py
@@ -81,8 +81,13 @@ trunk_url = ""
 addons_url = ""
 if grass_version != "unknown":
     major, minor, patch = grass_version.split(".")
-    trunk_url = f"https://github.com/OSGeo/grass/tree/releasebranch_{major}_{minor}/"
-    addons_url = f"https://github.com/OSGeo/grass-addons/tree/grass{major}/"
+    base_url = "https://github.com/OSGeo"
+    trunk_url = "{base_url}/grass/tree/releasebranch_{major}_{minor}/".format(
+        base_url=base_url, major=major, minor=minor
+    )
+    addons_url = "{base_url}/grass-addons/tree/grass{major}/".format(
+        base_url=base_url, major=major
+    )
 
 header_base = """<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>

